### ISSUE TITLE
change title of site

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/mbon-docs#4]
This site is documenting how to contribute to the MBON data portal, so we should update the title to clearly state that. 

Right now it's a little confusing having [this help site for the MBON portal](https://mbon.ioos.us/help/index.html) and [this MBON Portal documentation site](https://ioos.github.io/mbon-docs/). 

If we can merge the two together, that would be great! 

I think the sections [using the MBON portal](https://ioos.github.io/mbon-docs/#using-the-mbon-data-portal), [adding data to the map view](https://ioos.github.io/mbon-docs/#adding-to-the-map-view), and [how biodiversity products are created](https://ioos.github.io/mbon-docs/#how-biodiversity-data-products-are-created) should be in the [help site for the MBON portal](https://mbon.ioos.us/help/index.html#) as they are documenting specifics about the portal.

The site in this repository should be dedicated to providing documentation to the MBON projects on how to properly manage their biodiversity data. Not necessarily specific to the MBON portal. 

In short, let's change the title so it's clear what each page is documenting. Then, lets look at merging the documentation.

**Comment by MathewBiddle** [comment]
I propose we change the name of this page to 
### Marine Life Data and File Formatting

**Comment by MathewBiddle** [comment]
_Update_ change the title to:

# MBON Data and File Formatting

**Comment by MathewBiddle** [comment]
Now that some content has been added and the structure is being reworked, I'm wondering about the title again. Would this be better suited as 

# MBON Data Management and Cyberinfrastructure

I think that more appropriately described what documentation I would like to see in here. In the MBON Data Flow section we talk about ERDDAP, Darwin Core, sharing data to OBIS and NCEI, and a pointer for loading into MBON portal. So, does that title make sense?
